resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -62,49 +137,300 @@
+            "Action": [
+                "ec2:DescribeImageAttribute"
+            ],
+            "Resource": "arn:aws:ec2:*:*:image/*"
+        },
+        {
+            "Sid": "IdFormat",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePrincipalIdFormat",
+                "ec2:DescribeIdentityIdFormat",
+                "ec2:DescribeIdFormat",
+                "ec2:DescribeAggregateIdFormat"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AddressAttribute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddressesAttribute"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Addresses",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddresses"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AccountAttributes",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAccountAttributes"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "ByoipCidrs",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeByoipCidrs"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "CapacityReservation",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeCapacityReservations"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "CarrierGateways",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeCarrierGateways"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Instances",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeClassicLinkInstances"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "CustomerGateways",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeCustomerGateways"
+            ],
             "Resource": "*"
         },
         {
+            "Sid": "InstanceTypeOfferings",
             "Effect": "Allow",
             "Action": [
-                "s3:ListBucket"
+                "ec2:DescribeInstanceTypeOfferings"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "KeyPairs",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeKeyPairs"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "LaunchTemplate",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeLaunchTemplates"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "PlacementGroups",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePlacementGroups"
```

Comment:
What's this permission used for?

Reply:
Does not seem useful so will remove this permission